### ISSUE TITLE
xssproxy receives dbus messages for disabling screensaver but the screen still goes blank/start screensaver

### ISSUE THREAD

**Issue by fedxyz** [vincentbernat/xssproxy#5]
Hi,

I installed `xssproxy` on ubuntu, using it with verbose options I get some logs while firefox is playing a video like:

```
Inhibit request app=':1.2349' name='firefox' reason='audio-playing'
Disabling screensaver
Inhibit request app=':1.2349' name='firefox' reason='video-playing
```
That seems OK, but looking at `xset q` I still see `DPMS is Enabled` and the screen goes in standby after some time.
I tried with `xscreensaver`  installed and running but still the screen goes blank or the screensaver is started.

I am using `xssproxy version 1.0.0`.

I am using `i3` as window manager.

**Comment by vincentbernat** [comment]
xssproxy calls `XSuspendScreenSaver` which stops the timer. This is not visible in `xset q`. I am unable to reproduce. mpv does the same thing, can you run it with `-v` and check if it works for it? You should have the following line:
```
[vo/gpu/x11] Disabling screensaver.
```
It also has code to disable DPMS, but it should not be used if the first method succeeds. It will display "Setting DPMS" if it uses this code.

**Comment by fedxyz** [comment]
Hi!

sorry for the late reply in this time I tested `xssproxy` and I can say that it works most of the times, few times it didn't work but it's ok I was not able to investigate about this.
What I changed is that now I start `xssproxy` when my wm start (i3 in its config). 
When I opened the ticket I used to run it as for test via console for verbose option but I don't think this change its behaviour.

As a feature request, if possible, I would like to use `xssproxy` as a "client" too to query if the suspend was activated/deactivadted, to have a status like.
This could be useful imho as for test or to integrate it in status bar software like `i3blocks`.

Thanks, as for me the ticket can be closed.
